packages/desktop-client/src/components/accounts/TransactionsTable.test.js

```diff
@@ -8,22 +8,28 @@ import {
   generateTransaction,
   generateAccount,
   generateCategoryGroups,
-  TestProvider,
 } from 'loot-core/src/mocks';
+import { TestProvider } from 'loot-core/src/mocks/redux';
```

Comment:
This test was not being executed before, so needed to change a few things for it to start working